pkg/controller/secretsyncer/secretsyncer.go

```diff
@@ -162,17 +165,28 @@ func (r *reconciler) reconcile(ctx context.Context, log *logrus.Entry, encodedRe
 	}
 
 	var mirrorErrors []error
+	var conflictErrors []error
 	for _, mirrorConfig := range r.config().Secrets {
 		if mirrorConfig.From.Namespace != sourceSecretName.Namespace || mirrorConfig.From.Name != sourceSecretName.Name || (mirrorConfig.To.Cluster != nil && *mirrorConfig.To.Cluster != cluster) {
 			continue
 		}
 
 		// TODO: Ideally we would would create one reconcile.Request per target. This here means
 		// that if a single target is broken, it will send all targets into exponential backoff.
-		mirrorErrors = append(mirrorErrors, r.mirrorSecret(ctx, log, cluster, client, source, mirrorConfig.To.SecretLocation))
+		if err := r.mirrorSecret(ctx, log, cluster, client, source, mirrorConfig.To.SecretLocation); err != nil {
+			if apierrors.IsConflict(err) {
+				conflictErrors = append(conflictErrors, err)
+			} else {
+				mirrorErrors = append(mirrorErrors, err)
+			}
+		}
 	}
 
-	return utilerrors.NewAggregate(mirrorErrors)
+	// If we *only* hit conflict errors return a "representative" sample of them to pass a IsConflict check
+	if len(conflictErrors) > 0 && len(mirrorErrors) == 0 {
```

Comment:
This is not the prettyest way to do it. Ideally I'd like to track the "we only saw conflict errors" as a boolean and if true, then I'd wrap the aggregate in a new conflict error, but I have found no way how to easily construct an error that would pass `IsConflict`. The only way I found was to build a `StatusError` instance myself which I don't see is much better than what I have here.